examples/plot_credit_loan_decisions.py

```diff
@@ -0,0 +1,1293 @@
+    "balanced_acc_error": balanced_accuracy_error,
+    "selection_rate": selection_rate,
+    "false_positive_rate": false_positive_rate,
+    "false_positive_error": false_positive_error,
+    "false_negative_rate": false_negative_rate,
+    "false_negative_error": false_negative_error,
+}
+
+# %%
+# Select a subset of metrics to report to avoid information overload
+metrics_to_report = [
+    "balanced_accuracy",
+    "false_positive_rate",
+    "false_negative_rate",
+]
+
+# %%
+# To compute the disaggregated performance metrics, we will use the
+# :code:`MetricFrame` object within the Fairlearn library. We will pass in our
+# dictionary of metrics :code:`fairness_metrics`, along with our test labels :code:`y_test`
+# and test predictions :code:`Y_pred`. In addition, we pass in the
+# *sensitive_features* :code:`A_test` to disaggregate our model results.
+
+# %%
+# Instantiate the MetricFrame for the unmitigated model
+metricframe_unmitigated = MetricFrame(
+    metrics=fairness_metrics,
+    y_true=y_test,
+    y_pred=Y_pred,
+    sensitive_features=A_test,
+)
+
+metricframe_unmitigated.by_group[metrics_to_report]
+
+metricframe_unmitigated.difference()[metrics_to_report]
+
+metricframe_unmitigated.overall[metrics_to_report]
+
+
+# %%
+def plot_group_metrics_with_error_bars(metricframe, metric, error_name):
+    """
+    Plots the disaggregated metric for each group with an associated
+    error bar. Both metric and the error bar are provided as columns in the
+    provided MetricFrame.
+
+    Parameters:
+    - metricframe: The metricframe containing the metrics and their associated uncertainty quantification.
+    - metric: The set of metrics to plot
+    - error_name: The associated standard error for each metric in metric
+
+    Returns:
+    - Matplotlib Plot of point estimates with error bars
+    """
+    grouped_metrics = metricframe.by_group
+    point_estimates = grouped_metrics[metric]
+    error_bars = grouped_metrics[error_name]
+    lower_bounds = point_estimates - error_bars
+    upper_bounds = point_estimates + error_bars
+
+    x_axis_names = [
+        str(name) for name in error_bars.index.to_flat_index().tolist()
+    ]
+    plt.vlines(
+        x_axis_names,
+        lower_bounds,
+        upper_bounds,
+        linestyles="dashed",
+        alpha=0.45,
+    )
+    plt.scatter(x_axis_names, point_estimates, s=25)
+    plt.xticks(rotation=0)
+    y_start, y_end = np.round(min(lower_bounds), decimals=2), np.round(
+        max(upper_bounds), decimals=2
+    )
+    plt.yticks(np.arange(y_start, y_end, 0.05))
+    plt.ylabel(metric)
+
+
+# %%
+plot_group_metrics_with_error_bars(
+    metricframe_unmitigated, "false_positive_rate", "false_positive_error"
+)
+
+# %%
+plot_group_metrics_with_error_bars(
+    metricframe_unmitigated, "false_negative_rate", "false_negative_error"
+)
+
+# %%
+metricframe_unmitigated.by_group[metrics_to_report].plot.bar(
+    subplots=True, layout=[3, 1], figsize=[12, 18]
```

Comment:
I think it might look better if we replace this by the layout [1, 3] (with the appropriate figsize)? But I guess it slightly depends on whether we expect people to consume it on desktop or mobile. If we change it here, we should also change it elsewhere in the example.

Reply:
This is what it renders as right now. That looks fine to me(?) The screenshot is from my desktop, not mobile.

![image](https://user-images.githubusercontent.com/10245648/209249459-30d0f99a-cae4-4329-863e-aa17187df5b5.png)

Reply:
you also need to adjust figsize... i've implemented in my commit below, so this is done.